docs/Products/VerifiedAssetOracle/glossary.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 3
+description: Verified Asset Oracle Glossary
+keywords: [glossary, Verified Asset Oracle, VAO]
+---
+
+ # Glossary
+
+ ## General Key Concepts 
+
+**NAV (Net Asset Value)**
+Represents the total value of the investment fund. The value is calculated by subtracting its liabilities from its assets. 
+
+NAV = Assets - Liabilities
+
+
+**NAV/S (Net Asset Value per Share)**
+The value of a single share, calculated as the total Net Asset Value divided by the total number of shares.
+
+**AUM (Assets Under Management)**
+Represents the total market value of all assets managed by the fund held by the portfolio manager or investment company on behalf of its clients.
+
+**TVL (Total Value Locked)**
+Calculated as:
+TVL = NAV/S * Total Supply
+
+Reflects the total onchain value tied to the asset.
+
+**Yield**
+The earnings generated and realized on an investment over a period of time.
+
+**Token Price**
+ The price of the token on a given exchange or as an aggregated token price. It may differ from the price per share because the actual token price is determined by the exchange, not by the fund.
+ e.g.: the token price on [ByBit](https://www.bybit.com/en/price/superstate-short-duration-us-government-securities-fund-ustb/) and the NAV per share on the [Superstate](https://superstate.com/ustb#key-facts) itself for the same timestamp.
+
+**CUSIP (Committee on Uniform Securities Identification Procedures number)**
+It is a unique identifier for securities issued in the United States. A CUSIP is similar to an ISIN but does not include the country code. To convert a CUSIP into an ISIN, simply add the country prefix (e.g:"US").
+
+**Yield to maturity (YTM)** 
+The total return anticipated on a bond if the bond is held until it matures. 
+To compute [YTM](https://www.investopedia.com/terms/y/yieldtomaturity.asp), the following formula can be used:
+
+[(FaceValue - MarketValue)/MarketValue] x [365/DaysToMaturity]
```

Comment:
This formula looks different than the one on the linked page (line 41):

YTM = [ C+ (FV - PV) ÷ t ] ÷ [ (FV + PV) ÷ 2 ]

Where:
C = Coupon Payment
FV = Face Value
PV = Present Value/Current Price
t = Years to Maturity